curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65422ba173a18b1bedef1bb6.md

```diff
@@ -13,10 +13,10 @@ Inside the `if` block, call the `playSong()` function with the `id` of the first
 
 # --hints--
 
-You should create an `if` statement with the condition `userData?.currentSong === null`.
+You should create an `if` statement with the condition `userData?.currentSong`.
```

Comment:
This hint is incorrect.

We shouldn't be checking for if `userData?.currentSong` is truthy. 
We should just check if `userData?.currentSong` is falsey. That will allow for multiple correct answers instead of one.